gen/src/Gen/AST/Data/Syntax.hs

```diff
@@ -1,31 +1,26 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
+-- |
 -- Module      : Gen..AST.Data.Syntax
 -- Copyright   : (c) 2013-2021 Brendan Hay
--- License     : This Source Code Form is subject to the terms of
+-- License     : This Source Code Form is subject Lens.to the terms of
```

Comment:
Oops

Reply:
I've nuked all the module headers in the generator - since it's not uploaded to Hackage, it doesn't make a lot of sense - the license is already included in the repository root.

Reply:
Makes sense.